CHANGELOG.latest.md

```diff
@@ -1,49 +1,20 @@
-|  | v3 | v4 |
-| :-: | :-: | :-: |
-| iOS | 9.0 | 11.0 |
-| tvOS | 9.0 | 11.0 |
-| macOS | 10.12 | 10.13 |
-| watchOS | 6.2 | 6.2 (unchanged) |
-
-### StoreKit 2 support:
-- This beta introduces new methods that add functionality using StoreKit 2:
-    - `showManageSuscriptions(completion:)`
-    - `beginRefundRequest(forProduct:)`
-    - `beginRefundRequest(forEntitlement:)`. 
-    - `beginRefundRequestForActiveEntitlement()`
- - `checkTrialOrIntroductoryPriceEligibility(productIdentifiers:completion:)` now uses StoreKit 2 if it's available, to make calculation more accurate and fast.
- - A new flag has been introduced to `setup`, `useStoreKit2IfAvailable` (defaults to `false`), to use StoreKit 2 APIs for purchases instead of StoreKit 1.
+In addition to all of the changes from beta 9, beta 10 introduces the following updates:
 
-### `Async` / `Await` alternative APIs
-- In purchases-ios v3, `Async` / `Await` alternative APIs were made available through Xcode's auto-generation for Objective-C projects. This beta re-adds the `Async` / `Await` alternative APIs for v4.
-
-### New APIs:
-
-- `showManageSuscriptions(completion:)`: Use this method to show the subscription management for the current user. Depending on where they made the purchase and their OS version, this might take them to the `managementURL`, or open the iOS Subscription Management page. 
-- `beginRefundRequestForCurrentEntitlement`: Use this method to begin a refund request for the purchase that granted the current entitlement.
-- `beginRefundRequest(forProduct:)`: Use this method to begin a refund request for a purchase, specifying the product identifier.
-- `beginRefundRequest(forEntitlement:)`: Use this method to begin a refund request for a purchase, specifying the entitlement identifier.
-- Adds an optional `useStoreKit2IfAvailable` parameter to `setup` (defaults to `false`). If enabled, purchases will be done by using StoreKit 2 APIs instead of StoreKit 1. **This is currently experimental, and not all features are supported with StoreKit 2 APIs**.
-- Use `verboseLogHandler` or `verboseLogs` to enable more details in logs, including file names, line numbers and method names.
-
-### Known issues:
-- Promotional offers and deferred purchases are not currently supported with StoreKit 2. If your app uses either of those, you should omit `useStoreKit2IfAvailable` in `setup` or set it to `false`.
+### Breaking changes:
+- A new type, `StoreTransaction`, replaces `SKPaymentTransaction` in the return types of the following methods:
+    - `Purchases.purchase(product:completion:)`
+    - `Purchases.purchase(package:completion:)`
+    - `Purchases.purchase(package:discount:completion:)`
+    - `Purchases.purchase(package:discount:completion:)`
+    - `PurchasesDelegate.purchases(shouldPurchasePromoProduct:defermentBlock:)`
+    - `CustomerInfo.nonSubscriptionTransactions`
+- `StoreProduct.PromotionalOffer` has been renamed to `StoreProduct.StoreProductDiscount`.
 
 ### Other changes: 
```

Comment:
is it clear enough that this also means everything else that came from beta 9?

Reply:
Yup, but considering how big beta 8 was maybe add a line with a link to beta 8 changes? :)

Reply:
added!